### PR TITLE
Add README with dependencies/subrepo management instructions

### PR DESCRIPTION
**Pull request by alidiusk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+### Dependencies
+
+There are several dependencies required for the development and deployment of this project.
+The first is Docker Engine and Docker Compose. The following list links to the installation
+instructions for each of these:
+
+* [Docker Engine installation instructions.](https://docs.docker.com/engine/install/)
+* [Docker Compose installation instructions.](https://docs.docker.com/compose/install/linux/)
+
+<!-- TODO: identify and add other dependencies; Python? Rust? -->
```
Comment (TheCandianVendingMachine) [inline README.md:16]: Only Python and Docker are assumed to be dependencies. Everything will be managed in this with python, and Rust will only be launched in containers which will be downloaded on a per-container basis

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Potato Farmer
+
+Manages `potato_field` and `potato_plants`. Main way to administrate containers.
```
Comment (TheCandianVendingMachine) [inline README.md:3]: `potato_field` is the api gateway, so idk if "manages" makes the most sense here. I dont have any alternatives rn though

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Setup
+
+### Dependencies
+
+There are several dependencies required for the development and deployment of this project.
+The first is Docker Engine and Docker Compose. The following list links to the installation
+instructions for each of these:
+
+* [Docker Engine installation instructions.](https://docs.docker.com/engine/install/)
+* [Docker Compose installation instructions.](https://docs.docker.com/compose/install/linux/)
```
Comment (TheCandianVendingMachine) [inline README.md:14]: You linked the linux doc, most developers will probably be on windows

File: README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+<!-- TODO: identify and add other dependencies; Python? Rust? -->
+
+### Sub-repository management
+
+The tool `setup.py` can be used to clone (or symlink) project repositories for deployment.
+
+For instance, the following command will clone the `plant_dashboard` and `plant_missions` repositories
```
Comment (TheCandianVendingMachine) [inline README.md:22]: name them as they are named: `potato_plant missions` and `potato_plant_dashboard`. Likewise, indicate these are hosted in the BW organization

**Review by TheCandianVendingMachine — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alidiusk) [reply README.md:3]: Yeah, I just took this from the repo sidebar but I do feel similarly.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Potato Farmer
+
+Manages `potato_field` and `potato_plants`. Main way to administrate containers.
+
+## Setup
+
+### Dependencies
+
+There are several dependencies required for the development and deployment of this project.
+The first is Docker Engine and Docker Compose. The following list links to the installation
```
Comment (TheCandianVendingMachine) [inline README.md:10]: nitpick but “the first is docker compose” but never gets to the second

**Review by TheCandianVendingMachine — CHANGES_REQUESTED** [review]

**Review by TheCandianVendingMachine — APPROVED** [review]

**Pull request merged**